editor.planx.uk/src/components/Feedback/FeedbackForm.test.tsx

```diff
@@ -0,0 +1,147 @@
+import { waitFor } from "@testing-library/react";
+// eslint-disable-next-line no-restricted-imports
+import userEvent from "@testing-library/user-event";
```

Comment:
This is a restricted import for a reason - please see https://testing-library.com/docs/user-event/intro#writing-tests-with-userevent

You should still be able to access a `user` as an output of `setup()`

```ts
const { getByLabelText, getByText, user } = setup(
  <FeedbackForm
    inputs={mockUnlabeledInput}
    handleSubmit={mockHandleSubmit}
  />,
);

await user.type(
  getByLabelText("Leave your feedback"),
  "This is a test comment",
);
await user.click(getByText("Send feedback"));
```

Reply:
Nice one I'll check it out 👍 

I'd spotted it being overloaded elsewhere and incorrectly assumed we could ignore it 🙈 

I'll also update my other tests where I've done the same.

Reply:
Fixed here: d8d9e0ffeaab0a01a55db3cafc4916bdc076799f & 7a484a5f4644291235e76959953827aa84929712